{
    "object_alias": "exface.Core.MESSAGE",
    "columns": [
        {
            "name": "CREATED_ON",
            "attribute_alias": "CREATED_ON"
        },
        {
            "name": "MODIFIED_ON",
            "attribute_alias": "MODIFIED_ON"
        },
        {
            "name": "CREATED_BY_USER",
            "attribute_alias": "CREATED_BY_USER"
        },
        {
            "name": "MODIFIED_BY_USER",
            "attribute_alias": "MODIFIED_BY_USER"
        },
        {
            "name": "UID",
            "attribute_alias": "UID"
        },
        {
            "name": "APP",
            "attribute_alias": "APP"
        },
        {
            "name": "CODE",
            "attribute_alias": "CODE"
        },
        {
            "name": "TITLE",
            "attribute_alias": "TITLE"
        },
        {
            "name": "DESCRIPTION",
            "attribute_alias": "DESCRIPTION"
        },
        {
            "name": "HINT",
            "attribute_alias": "HINT"
        },
        {
            "name": "TYPE",
            "attribute_alias": "TYPE"
        },
        {
            "name": "DOCS",
            "attribute_alias": "DOCS"
        }
    ],
    "rows": [
        {
            "CREATED_ON": "2016-12-20 07:55:40",
            "MODIFIED_ON": "2016-12-20 07:55:40",
            "CREATED_BY_USER": "0x31000000000000000000000000000000",
            "MODIFIED_BY_USER": "0x31000000000000000000000000000000",
            "UID": "0x11e6c689a72e43e9a3e40205857feb80",
            "APP": "0x11e6951dda3ba520b48b847beb4a5184",
            "CODE": "6T4RAVX",
            "TITLE": "Failed to create HTTP client",
            "DESCRIPTION": "",
            "HINT": "",
            "TYPE": "ERROR",
            "DOCS": ""
        },
        {
            "CREATED_ON": "2017-04-26 09:47:00",
            "MODIFIED_ON": "2017-08-22 15:22:58",
            "CREATED_BY_USER": "",
            "MODIFIED_BY_USER": "",
            "UID": "0x11e72a654b1c523f8f0674e5434dc47d",
            "APP": "0x11e6951dda3ba520b48b847beb4a5184",
            "CODE": "6UY4L3E",
            "TITLE": "Incorrect request to a webservice",
            "DESCRIPTION": "This error results from an incorrect request to a webservice. Please contact the technical support. Don't forget to communicate the error code!",
            "HINT": "",
            "TYPE": "ERROR",
            "DOCS": ""
        },
        {
            "CREATED_ON": "2017-04-26 10:51:55",
            "MODIFIED_ON": "2017-08-22 15:31:36",
            "CREATED_BY_USER": "",
            "MODIFIED_BY_USER": "",
            "UID": "0x11e72a6e5cb4394d8f0674e5434dc47d",
            "APP": "0x11e6951dda3ba520b48b847beb4a5184",
            "CODE": "6UY5F5D",
            "TITLE": "Authentication to a webservice failed",
            "DESCRIPTION": "The authentication to a webservice failed. You're not allowed to access the requested resource. Please contact the technical support. Don't forget to communicate the error code!",
            "HINT": "",
            "TYPE": "ERROR",
            "DOCS": ""
        },
        {
            "CREATED_ON": "2017-04-26 14:26:17",
            "MODIFIED_ON": "2017-08-22 15:21:49",
            "CREATED_BY_USER": "",
            "MODIFIED_BY_USER": "",
            "UID": "0x11e72a8c4ee7c3178f0674e5434dc47d",
            "APP": "0x11e6951dda3ba520b48b847beb4a5184",
            "CODE": "6UY86E7",
            "TITLE": "Internal error in webservice",
            "DESCRIPTION": "This error results from an internal error in a webservice. Please contact the technical support. Don't forget to communicate the error code!",
            "HINT": "",
            "TYPE": "ERROR",
            "DOCS": ""
        },
        {
            "CREATED_ON": "2017-04-26 14:31:07",
            "MODIFIED_ON": "2017-08-22 15:19:47",
            "CREATED_BY_USER": "",
            "MODIFIED_BY_USER": "",
            "UID": "0x11e72a8cfb8b80de8f0674e5434dc47d",
            "APP": "0x11e6951dda3ba520b48b847beb4a5184",
            "CODE": "6UY88MN",
            "TITLE": "Unknown error during request to a webservice",
            "DESCRIPTION": "An unknown error has occured during a request to a webservice. Please contact the technical support. Don't forget to communicate the error code!",
            "HINT": "",
            "TYPE": "ERROR",
            "DOCS": ""
        },
        {
            "CREATED_ON": "2017-04-26 15:43:00",
            "MODIFIED_ON": "2017-08-22 15:20:43",
            "CREATED_BY_USER": "",
            "MODIFIED_BY_USER": "",
            "UID": "0x11e72a9706685b938f0674e5434dc47d",
            "APP": "0x11e6951dda3ba520b48b847beb4a5184",
            "CODE": "6UY95WR",
            "TITLE": "No response from webserver",
            "DESCRIPTION": "This error occurs if a webserver can't be contacted for some reason. Please contact the technical support. Don't forget to communicate the error code!",
            "HINT": "",
            "TYPE": "ERROR",
            "DOCS": ""
        },
        {
            "CREATED_ON": "2017-05-04 08:13:08",
            "MODIFIED_ON": "2017-08-22 15:21:23",
            "CREATED_BY_USER": "",
            "MODIFIED_BY_USER": "",
            "UID": "0x11e730a182a8a06fb40774e5434dc47d",
            "APP": "0x11e6951dda3ba520b48b847beb4a5184",
            "CODE": "6V27IZ3",
            "TITLE": "The webservice could not be contacted",
            "DESCRIPTION": "This error occurs when a webservice could not be contacted at the URL specified in the current connection. Please contact the technical support. Don't forget to communicate the error code!",
            "HINT": "",
            "TYPE": "ERROR",
            "DOCS": ""
        },
        {
            "CREATED_ON": "2018-12-12 11:12:52",
            "MODIFIED_ON": "2018-12-12 11:23:56",
            "CREATED_BY_USER": "0x31000000000000000000000000000000",
            "MODIFIED_BY_USER": "0x31000000000000000000000000000000",
            "UID": "0x11e8fdfed8ded899a23ee4b318306b9a",
            "APP": "0x11e6951dda3ba520b48b847beb4a5184",
            "CODE": "73G87II",
            "TITLE": "Cannot convert oData 2.0 association to a relation in the metamodel",
            "DESCRIPTION": "",
            "HINT": "Try to re-run the model builder for the objects at both ends of the association node",
            "TYPE": "ERROR",
            "DOCS": "exface\/UrlDataConnector\/Docs\/OData\/The_metamodel_for_OData_2.0.md"
        },
        {
            "CREATED_ON": "2019-04-23 15:30:16",
            "MODIFIED_ON": "2019-04-23 15:42:34",
            "CREATED_BY_USER": "0x31000000000000000000000000000000",
            "MODIFIED_BY_USER": "0x31000000000000000000000000000000",
            "UID": "0x11e965dc7ff50a18812a0205857feb80",
            "APP": "0x11e6951dda3ba520b48b847beb4a5184",
            "CODE": "75C7YOQ",
            "TITLE": "Missing parameter for remote function call",
            "DESCRIPTION": "",
            "HINT": "Please include a column named exactly as the parameter in the input data (e.g. by using an input_mapper)!",
            "TYPE": "ERROR",
            "DOCS": ""
        }
    ],
    "totals_rows": [],
    "filters": {
        "operator": "AND",
        "conditions": [
            {
                "expression": "APP",
                "comparator": "=",
                "value": "0x11e6951dda3ba520b48b847beb4a5184",
                "object_alias": "exface.Core.MESSAGE"
            }
        ]
    },
    "rows_limit": null,
    "rows_offset": 0,
    "sorters": [
        {
            "attribute_alias": "CREATED_ON",
            "direction": "ASC"
        },
        {
            "attribute_alias": "UID",
            "direction": "ASC"
        }
    ]
}